includes/class-wpcom-legacy-redirector.php

```diff
@@ -92,9 +101,10 @@ static function maybe_do_redirect() {
 	/**
 	 * Enqueue the JS that builds the link previews.
 	 *
+	 * @param string $hook Get the current page hook.
+	 *
 	 * @since 1.4.0
 	 *
-	 * @param string $hook Get the current page hook.
```

Comment:
`@since` tag should be first - there's no reason to change this.

See https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

Reply:
I did not make this change manually. I applied the WP code style from PHP Storm which formatted accordingly. Do you have a more refined rule set I can load? The field guide did not have one to download. @GaryJones

Reply:
Doc block formatting was cleaned up and filled in where needed.